backend/src/controllers/admin/glossary.js

```diff
@@ -12,7 +12,8 @@ const path = require("path");
  * @returns {object}
  */
 const getAll = async (request, reply) => {
-  controller.userRequires(request, what, "admin_form_read_all");
+  controller.userRequires(request, "PMO-Manager-Edit-Capability", reply);
```

Comment:
I think the "PMO-Manager-Edit-Capability" should be stored in a constant somewhere. If we ever had to change the name of this we'd have to do it in like 50 places.